src/reset-password/controller/redirect.ts

```diff
@@ -0,0 +1,48 @@
+import Controller from '@curveball/controller';
```

Comment:
FIlenames should be corrected.. 'redirect' is not really what this controller does...